### PR TITLE
PUB 1834 - E2E Test for User Management

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-1834

### Change description

Added testing for the User Management page. 

Removed old tests which are now duplicated by this.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/end-to-end/tests/system-admin/system-admin-user-management-test.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    I.fillField('#email', testEmailAddress);
+    I.click('Apply filters');
+    I.see(testEmailAddress);
+    I.click('#manage-link');
+    I.see('Manage ' + testEmailAddress);
+    I.click('Change');
+    I.selectOption('#updatedRole', 'CTSC Admin');
+    I.click('Continue');
+    I.see('User Updated');
+    I.see('This user has been updated to a CTSC Admin');
```
Comment (ashwini-mv) [inline src/test/end-to-end/tests/system-admin/system-admin-user-management-test.ts:29]: When the text is result of an action then first time we use I.waitForText
  I.click('Continue');
    I.see('User Updated');
    I.see('This user has been updated to a CTSC Admin');

File: src/test/end-to-end/tests/custom-steps.ts [hunk 2]
```diff
@@ -37,5 +37,19 @@ export = function () {
...
+            this.fillField('#lastName', lastName);
+            this.fillField('#emailAddress', email);
+            this.click(role);
+            this.click('Continue');
+            this.see('Check account details');
+            this.click('Confirm');
+            this.see('Account has been created');
+        },
     });
 };
```
Comment (ashwini-mv) [inline src/test/end-to-end/tests/custom-steps.ts:55]: Is it possible to create and delete user through api call ?

**Review by ashwini-mv — CHANGES_REQUESTED** [review]

File: src/test/end-to-end/tests/system-admin/system-admin-user-management-test.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    I.click('#card-user-management');
+    I.see('User Management');
+
+    I.see('Next');
+    I.see('2 of');
+    I.click('.govuk-pagination__link');
+
+    I.see('Previous');
+    I.see('1 of');
+    I.click('.govuk-pagination__link');
```
Comment (KianKwa) [inline src/test/end-to-end/tests/system-admin/system-admin-user-management-test.ts:116]: Are we supposed to test "I.see('Next')" again after clicking the link?

File: src/test/end-to-end/functional.conf.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ setHeadlessWhen(testConfig.TestHeadlessBrowser);
 
 export const config: CodeceptJS.MainConfig = {
     name: 'functional',
-    tests: './tests/*-test.ts',
+    tests: './tests/**/*-test.ts',
```
[the remark this answers is not in the document]
Comment (ChrisS1512) [reply src/test/end-to-end/functional.conf.ts:10]: Good spot, updated

**Review by KianKwa — DISMISSED** [review]

**Review by ashwini-mv — APPROVED** [review]

**Review by KianKwa — APPROVED** [review]

**Review by ashwini-mv — APPROVED** [review]

**Pull request merged**